lib/linguist/languages.yml

```diff
@@ -5897,6 +5878,35 @@ Windows Registry Entries:
   codemirror_mode: properties
   codemirror_mime_type: text/x-properties
   language_id: 969674868
+Wolfram Language:
+  type: programming
+  extensions:
+  - ".cdf"
+  - ".m"
+  - ".ma"
+  - ".mt"
+  - ".nb"
+  - ".nbp"
+  - ".wl"
+  - ".wls"
```

Comment:
Since you added this file extension, could you fill in the `I am associating a language with a new file extension.` part of the template?